.github/workflows/deploy-formstack.yml

```diff
@@ -9,6 +10,7 @@ on:
     paths:
       - 'formstack-baton-requests/**'
       - '.github/workflows/deploy-formstack.yml'
+  workflow_dispatch:
```

Comment:
it looks like this manual trigger button doesn't show up until you merge the PR?